webdriver-spec.html

```diff
@@ -3342,14 +3344,9 @@
    <dt><var>id</var> <a>represents a web element</a>
    <dd>
     <ol>
-     <li><p>Let <var>element result</var> be the result of
-      <a>getting a known element</a> by
-      <a>UUID</a> reference <var>id</var>.
-
-     <li><p>If <var>element result</var> is a <a>success</a>,
-      let <var>element</var> be <var>element result</var>’s data.
-
-      <p>Otherwise, return <var>element result</var>.
+     <li><p>Let <var>element</var> be the result
+      of <a>trying</a> to <a>get a known element</a>
+      by <a>UUID</a> reference <var>id</var>.
```

Comment:
There is no requirement that a remote end use a UUID as the reference to an element. Raised #681